content/post/internal-boot2root-writeup.md

```diff
@@ -0,0 +1,543 @@
+---
+title: "Internal Boot2Root Write-Up"
+date: 2020-12-05T00:00:00-04:00
+author: "Nate Catelli"
+tags: ["ctf", "boot2root", "hacking", "writeup", "tryhackme"]
+description: "A boot2root writeup of the Internal host from TryHackMe"
+type: "post"
+draft: false
+---
+
+## Introduction:
+The Internal host took almost 24 hours to complete due to the sheer number of pivots required to complete it. Unlike many of the other boot2roots I've completed on THM, this host required a significant amount of review and manual poking around on the host above and beyond the results of automated enumeration tools like linPEAS. I enjoyed this host immensely and I thought it was incredibly brilliant machine. 
+
+## Environment
+The attack takes place on a flat network consisting of the attack host, a freshly-booted Kali Linux livecd, and the target host. Information about the host was extremely limited other than that there would be two flags available corresponding to a user and root flag. It was also known that the host contained a web application, and that the host, known by the domain internal.thm, was the only host in scope.
+
+## Attack 
+Prior to starting the attack, I prepared my workstation by setting up burpsuite, installing the certificates in firefox and defining the scope to include only the target host. I also opened msfconsole and configured it to connect to a postgres backend of msfdb. I also installed `jq`, [gobuster](https://github.com/OJ/gobuster) and the [seclists](https://github.com/danielmiessler/SecLists) wordlist collections. Finally I added `internal.thm` to my hosts file mapped to the target IP per the provide scope document.
+
+### Host enumeration
+I started the attack by running an SYN and version and OS scan against the host which identified only 2 listening ports on ssh and http while also confirming that this host was running linux.
```

Comment:
by running SYN, version, and OS scans against the host

which identified 2 open ports for ssh and http, and confirmed that the host was running linux